### ISSUE TITLE
Sweater description in the Grab Alert (when depleated of negative charges), not correct

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#300]
Just noticed in Version 1.4.0-dev.4 that the sweater's description in the Grab Alert is not correct once negative charges are depleted: 

In Grab Alert:
"Sweater has all more positive charges than negative charges."

Grab Alert in this case should be:
"Sweater has no more negative charges, only positive charges."

Full Grab Alert example for this case:
"Grabbed. On right arm of sweater. Has many more negative charges than positive charges. Sweater has no negative charges, only positive charges. Press W,A,S,or D key to move balloon. Space key to release."

@jessegreenberg, this might already be on your radar as you add in the "Finding more charges" alerts.

**Comment by jessegreenberg** [comment]
Understood, thanks @terracoda haven't quite gotten there yet.

**Comment by jessegreenberg** [comment]
Looks like this has been fixed in other issues, but we are missing a period at the end of the grabbed alert.

**Comment by jessegreenberg** [comment]
But only when we are on the sweater.

**Comment by jessegreenberg** [comment]
The above commit fixes all the punctuation issues I could find, including this one. Closing.
